viewing_party/party.py

```diff
@@ -1,23 +1,152 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    new_movie = {}
+    new_movie["title"] = title
+    new_movie["genre"] = genre
+    new_movie["rating"] = rating
+    
+    if None in new_movie.values():
+        return None
+    else:
+        return new_movie
+
+def add_to_watched(user_data, movie):
+     user_data["watched"].append(movie)
+     return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(user_data, title):
+    # loop through dict of watch list
+    for movie in user_data["watchlist"]:
+        if movie["title"] == title:
+            user_data["watched"].append(movie)
+            user_data["watchlist"].remove(movie)    

```

Comment:
It seems counterintuitive, but we could add a second return statement here within the if block (same return statement as below). This will just stop the function as soon as it finds the movie we're looking for and keep it from running through the rest of the loop.